### PR TITLE
Remove use of skip_fork_db flag

### PR DESCRIPTION
**Pull request by jmjatlanta**
Possible fix for #1313 

skip_fork_db removed, as it is barely used

### CODE REVIEW

File: libraries/chain/db_block.cpp [hunk 1]
```diff
@@ -129,77 +129,74 @@ bool database::push_block(const signed_block& new_block, uint32_t skip)
 bool database::_push_block(const signed_block& new_block)
 { try {
    uint32_t skip = get_node_properties().skip_flags;
-   if( !(skip&skip_fork_db) )
-   {
-      /// TODO: if the block is greater than the head block and before the next maitenance interval
-      // verify that the block signer is in the current set of active witnesses.
+   /// TODO: if the block is greater than the head block and before the next maitenance interval
```
Comment (pmconrad) [inline libraries/chain/db_block.cpp]: Please fix the typo while you're at it.

Comment (jmjatlanta) [reply libraries/chain/db_block.cpp]: Are you speaking of the three slashes? If not, I am blind.

Comment (abitmore) [reply libraries/chain/db_block.cpp]: "maitenance"

Comment (jmjatlanta) [reply libraries/chain/db_block.cpp]: Yep, I just had my eyes checked too. Maybe their equipment needs calibration. Typo fixed.

**Pull request merged**